servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultLoadBalancer.java

```diff
@@ -90,6 +90,8 @@ final class DefaultLoadBalancer<ResolvedAddress, C extends LoadBalancedConnectio
     private static final long RESUBSCRIBING = -1L;
 
     private volatile long nextResubscribeTime = RESUBSCRIBING;
+    @Nullable
```

Comment:
In reality this is never null after initialization, much like every other field. Could we ignore it, or give it a non-null initial value that we just discard?

Reply:
We only need to check for "current". So, it doesn't matter if it's nullable or not because we don't access any fields or methods of `currentSubscriber` -> NPE is not possible. 
To make it non-null I will have to add a `NoopSubscriber` in every implementation, which seems like a boilerplate for this use-case.

Reply:
I was thinking you could give it an idle `new EventSubscriber()` during the declaration but I'll admit that has its own drawbacks so I suspect you're right that it's fine to just give it `@Nullable` for now.